### ISSUE TITLE
Focus ring showing on every tool bar button in Kate

### ISSUE THREAD

**Issue by xsrvmy** [tsujan/Kvantum#536]
When using a kvantum theme, the focus ring for buttons always show in the kate toolbar, when the window is focused. This is not a problem with any of the built-in KDE themes.

**Comment by xsrvmy** [comment]
I think the relevant control is the toolbutton

**Comment by tsujan** [comment]
What do you mean by "focus ring". There's no focus rectangle on Kate's toolbar. Please attach a screenshot.

**Comment by tsujan** [comment]
Oh, if you mean Kate's *status-bar*, that's a bug in Kate's style code, for which I have no intention to include a workaround in Kvantum.

KDE devs should fix it but they won't because they don't see it with Breeze and most other styles don't have focus rectangles either. But, fortunately, `plastique` does (`plastique` belongs to `qt5-styleplugins` in Arch). If you have `plastique`, you could see the mess by running `kate -style plastique` in a terminal and move the mouse over Kate's status-bar.
